__tests__/apiCategories.test.js

```diff
@@ -39,5 +39,45 @@ describe('APIs', () => {
     });
   });
   ////
-
+  describe('4. GET /api/reviews/:review_id', () => {
+    test('status:200 OK, responds with review object with required properties', () => {
+      return request(app)
+        .get('/api/reviews/1')
+        .expect(200)
+        .then(({ body }) => {
+          const { reviewObj } = body;
+          expect(reviewObj).toBeInstanceOf(Object);
+          expect(Object.keys(reviewObj)).toHaveLength(9);
+          expect(reviewObj).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
+              title: expect.any(String),
+              category: expect.any(String),
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_body: expect.any(String),
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            })
+          );
+        });
```

Comment:
Hey Iliya! Here we are expecting a very specific review (review 1 to be exact) so you can test for actual values rather than the data types of each key as you have here. This will make this test more robust and specific to this case.